### DISCUSSION TITLE
[27 marzo 2023] Automi

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/automi-calcolabilita-complessita#7] [Automi]
![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/69389461/8b73c371-587b-4e64-a391-367ef9d5c7ce)

**Comment by Exyss** [comment]
Il primo punto è un caso particolare dell'esercizio #16

La dimostrazione risulta per tanto identica, basta sostituire $k$ con $3$

----

La dimostrazione richiesta nel secondo punto è presente sul libro di testo

Per quanto riguarda l'esempio di applicazione, riporto l'esempio presente sui miei appunti:

![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/65043207/e11d7bef-7b49-4a6d-8361-458cb7953584)
